web/public/mgmt/config.yml

```
@@ -2516,6 +2516,64 @@ collections:
           - { label: "school/course", value: "school-course" }
           - { label: "object or vehicle", value: "object-vehicle" }
 
+  - name: "raffle-bingo-steps"
+    label: "Raffle Bingo Steps"
+    folder: "content/src/roadmaps/raffle-bingo-steps"
+    summary: "{{displayname}} - {{name}}"
+    identifier_field: "{{filename}}"
+    slug: "{{fields.filename}}"
+    delete: true
+    sortable_fields: ["id", "name", "{{urlSlug}}", "industryId"]
+    create: true
+    editor:
+      preview: true
+    fields:
+      - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
+      - {
+          label: "Id",
+          name: "id",
+          widget: "write-once-read-only-no-space",
+          required: true,
+          hint: "This is should be the same as the filename.",
+        }
+      - {
+          label: "Filename",
+          name: "filename",
+          widget: "write-once-read-only-no-space",
+          hint: "This is the name of the file in the code only.",
+        }
+      - {
+          label: "Display Name",
+          name: "displayname",
+          widget: "nospace",
+          required: true,
+          hint: "This the name displayed internally in CMS list.",
+        }
+      - {
+          label: "Url Slug",
+          name: "urlSlug",
+          widget: "nospace",
+          hint: "Lowercase, no spaces, add hyphens between words.",
+        }
+      - { label: "Title (Navigator)", name: "name", widget: "string" }
+      - {
+          label: "Summary Description (Webflow and Navigator)",
+          name: "summaryDescriptionMd",
+          widget: "markdown",
+        }
+      - {
+          label: "Call To Action Text (Webflow & Navigator)",
+          name: "callToActionText",
+          widget: "string",
+          required: false,
+        }
+      - {
+          label: "Call To Action Link (Webflow & Navigator)",
```

[sand] Given this is never going to show up in Webflow, we don't need to make a distinction about which application content is applicable to.
```suggestion
      - { label: "Title", name: "name", widget: "string" }
      - {
          label: "Summary Description",
          name: "summaryDescriptionMd",
          widget: "markdown",
        }
      - {
          label: "Call To Action Text",
          name: "callToActionText",
          widget: "string",
          required: false,
        }
      - {
          label: "Call To Action Link",
```